worker_node/job/send.js

```diff
@@ -0,0 +1,149 @@
+const Base = require('./base');
+const Ain = require('@ainblockchain/ain-js').default;
+const delay = (time) => new Promise(resolve => setTimeout(resolve, time));
+const BLOCK_TIME = 8000;
+
+class Send extends Base {
+  #ain;
+
+  constructor(config) {
+    super(config);
+    this.output = {
+      message: '',
+      statistics: {
+        success: 0,
+        fail: 0,
+        timeFromStartToFinish: 0,
+      },
+      txHashList: [],
+      startBlockNumber: 0,
+      finishBlockNumber: 0,
+    };
+    this.#ain = new Ain(this.config.ainUrl);
+    this.#ain.wallet.add(this.config.ainPrivateKey);
+    this.#ain.wallet.setDefaultAccount(this.config.ainAddress);
+    this.#ain.provider.setDefaultTimeoutMs(60 * 1000);
+  }
+
+  async getRecentBlockInformation(keyList) {
+    try {
+      const information = await this.#ain.provider.send('ain_getRecentBlock');
+      return keyList.reduce((acc, cur) => {
+        acc[cur] = information[cur];
+        return acc;
+      }, {});
+    } catch (err) {
+      console.log(`Error while getRecentBlockInformation (${err.message})`);
+      throw err;
+    }
+  }
+
+  async initPermission() {
+    const path = this.config.baseTx.operation.ref;
+    const setOwnerTx = {
+      operation: {
+        type: 'SET_OWNER',
+        ref: path,
+        value: {
+          '.owner': {
+            owners: {
+              '*': {
+                write_owner: true,
+                write_rule: true,
+                write_function: true,
+                branch_owner: true,
+              },
+            },
+          },
+        },
+      },
+      nonce: -1,
+    };
+    const setRuleTx = {
+      operation: {
+        type: 'SET_RULE',
+        ref: path,
+        value: {
+          '.write': true,
+        },
+      },
+      nonce: -1,
+    };
+    await this.#ain.sendTransactionBatch([
+      setOwnerTx,
+      setRuleTx,
+    ]);
+
+    await delay(2 * BLOCK_TIME);
+
+    const writePermission = await this.#ain.db.ref(path).evalRule({value: null});
+    if (!writePermission) {
+      throw Error(`Can't write database (permission)`);
+    }
+  }
+
+  async sendTxs() {
+    const delayTime = this.config.time / this.config.number * 1000;
```

Comment:
The meaning of variable names "number", "time" is ambiguous.
Suggests variable names such as "numberOfTransactions" and "duration".

Reply:
Looks better. Done!